abdulfatai360/book-finder
# Add Custom Icons to Project

#### What does this PR do?
Add favicons to the project

#### How should this be manually tested?
- Clone this repo: ```https://github.com/abdulfatai360/book-finder.git```
- Navigate to project folder ```book-finder/```
- Open the ```index.html``` file on a browser
- A custom icon should appear on the title bar of the browser's tab
